### PR TITLE
make demo work in linux

### PR DESCRIPTION
**Pull request by yuval-k**
BOT NOTES: 
resolves https://github.com/solo-io/service-mesh-hub/issues/692

### CODE REVIEW

File: ci/setup-kind.sh [hunk 1]
```diff
@@ -16,15 +16,15 @@
...
 fi
 
 make clean
 
-# generate 16-character random suffix on these names
-managementPlane=management-plane-$(xxd -l16 -ps /dev/urandom)
-remoteCluster=target-cluster-$(xxd -l16 -ps /dev/urandom)
+# generate 1 allows to make several envs in parallel
+managementPlane=management-plane-$1
+remoteCluster=target-cluster-$1
```
Comment (GrahamGoudeau) [inline ci/setup-kind.sh:27]: why these changes?

Comment (yuval-k) [reply ci/setup-kind.sh:27]: xxd is not available on linux
